Assets/Scripts/AudioManager.cs

```diff
@@ -4,10 +4,20 @@
 
 public class AudioManager : MonoBehaviour
 {
+    public static AudioManager Instance { get; private set; }
     public Sound[] sounds;
-    // Start is called before the first frame update
     void Awake()
     {
+        if (Instance == null)
```

Comment:
is there a reason why you set up the audio manager to not be destroyed on load?

Reply:
The youtube tutorial said so

Reply:
lol fair enough